Release 0.48.0 (#1539)
packit/packit: CHANGELOG.md

# 0.48.0

- `packit source-git update-dist-git` and `packit source-git update-source-git` now check the synchronization of source-git and dist-git repositories prior to doing the update. If the update can't be done, for example, because the histories have diverged, the command provides instructions on how to synchronize the repositories. A `--force` option is available to try to update the destination repository anyway.

> - Downstream synchronization for the packit yaml configuration file should be fixed. (#1532)

Reviewer: ```suggestion
- Downstream synchronization of the Packit configuration file (aka `packit.yaml`) should be fixed. (#1532)
```
